### ISSUE TITLE
Steam Installer Requires permission changes to work

### ISSUE THREAD

**Issue by creativiii** [ValveSoftware/steam-for-linux#5783]
#### My system information

* Steam client version (build number or date): N/A
* Distribution (e.g. Ubuntu): Ubuntu 18.04.01 LTS
* Opted into Steam client beta?: N/A
* Have you checked for system updates?: N/A

#### Please describe your issue in as much detail as possible:
While this isn't a huge problem and is easy to fix by users that know what they're doing, new Linux users might be confused by it. I suppose that since it's such a minor problem it hasn't been reported officially, or at least I was not able to find anything about it here.

When downloading the .deb Steam file from [here](https://store.steampowered.com/about/) and running it, the installer does not work. If started by double clicking, Ubuntu gives the following error:

> steam could not setup steam data

If ran in terminal this is the output:

```
steam
Repairing installation, linking /home/leonardo/.steam/steam to /home/leonardo/.local/share/Steam
rm: cannot remove '/home/leonardo/.steam/steam': Is a directory
Setting up Steam content in /home/leonardo/.local/share/Steam
rm: cannot remove '/home/leonardo/.steam/steam': Is a directory
```
The current fix is changing permissions like so:

```
mv ~/.steam/steam/* ~/.local/share/Steam/
rmdir ~/.steam/steam
ln -s ../.local/share/Steam ~/.steam/steam
rm -rf ~/.steam/bin
```
This has been a bug since at least 2014, [here's](https://askubuntu.com/questions/470436/steam-cannot-set-up-steam-data) a bunch of people also having the same problem.

#### Steps for reproducing this issue:

1. Download Steam from [here](https://store.steampowered.com/about/) 
2. Start Steam installation by running the downloaded file

**Comment by kisak-valve** [comment]
Hello @lovemoebius, at some point the Ubuntu-provided steam package was run on your system. Ubuntu inherits its steam package from Debian and the Debian/Ubuntu provided steam package sets up a conflicting folder layout on first run of Steam.

Take this issue up with the Ubuntu or Debian package maintainer for steam.

**Comment by lovemoebius** [comment]
> Hello @lovemoebius, at some point the Ubuntu-provided steam package was run on your system. Ubuntu inherits its steam package from Debian and the Debian/Ubuntu provided steam package sets up a conflicting folder layout on first run of Steam.
> 
> Take this issue up with the Ubuntu or Debian package maintainer for steam.

I did first install it from the Software Center and then uninstalled it to test something.
Nevermind I guess, thank you for the quick response

**Comment by kisak-valve** [comment]
Hello @creativiii, are you still experiencing this issue on an up to date system?

This specific folder layout conflict should have been resolved in the Debian-modified Steam package 1.0.0.59-1 and made its way downstream to Ubuntu. Unfortunately, the version distributed by Ubuntu 18.04 is older than that.

**Comment by creativiii** [comment]
I haven't had problems since posting this so I assume it must be fixed! 😄
